### PR TITLE
[Housekeeping] Issues from first testings

### PR DESCRIPTION
**Pull request by coding46**
[WIP] Don't merge for now

### CODE REVIEW

**Comment by NamelessCoder** [comment]
Happy with this one, @coding46 - looks okay to me?

**Comment by coding46** [comment]
@NamelessCoder Yep.

File: Classes/Controller/AbstractBackendController.php [hunk 1]
```diff
@@ -331,13 +331,15 @@ protected function deepenDottedArray(array $array) {
...
 	 */
 	public function trimLanguageWrappersFromPostedData($post, $level = 0) {
-		foreach ($post as $name => $value) {
-			if ($name === 'vDEF' || $name === 'lDEF') {
-				return $this->trimLanguageWrappersFromPostedData($value, $level + 1);
-			} elseif (TRUE === is_array($value)) {
-				$post[$name] = $this->trimLanguageWrappersFromPostedData($value, $level + 1);
-			}
-		}
+        if (TRUE === is_array($post)) {
```
Comment (NamelessCoder) [inline Classes/Controller/AbstractBackendController.php]: Indent level/type looks wrong here. Return statement is further indented than foreach block?

**Comment by coding46** [comment]
@NamelessCoder Indentation fixed.

**Pull request merged**